### PR TITLE
AT_014.03 Entity > Chevron > Edit

### PR DESCRIPTION
**Pull request by EvgeniyaPiskunova**
https://trello.com/c/bIF3WwnH/420-tc01403-entity-chevron-edit
https://trello.com/c/GbGkE39C/77-us01403-entity-chevron-edit
https://trello.com/c/dDeI9TRE/446-at01403-entity-chevron-edit

### CODE REVIEW

**Review by OlenaSMarchenko — APPROVED** [review]

File: src/test/java/EntityChevronTest.java [hunk 1]
```diff
@@ -310,4 +311,33 @@ public void testCreateRecordSent2(){
         WebElement actualResult2 = getDriver().findElement(By.xpath("//tr/td[2]"));
         Assert.assertEquals(actualResult2.getText(),expectedResult);
     }
+
+    @Test
+    public void testChevronDeletePermanentlyRecord() {
+        TestUtils.scrollClick(getDriver(), getDriver().
+                findElement(By.xpath("//p[text()=' Chevron ']")));
+
+        By createButtonLocator = By.xpath("//i[text() = 'create_new_folder']");
```
Comment (AlexSharon) [inline src/test/java/EntityChevronTest.java:320]: you could also optimize code by using already existing locator:
private static final By CREATE_NEW_FOLDER = By.xpath("//i[text()='create_new_folder']");

File: src/test/java/EntityChevronTest.java [hunk 2]
```diff
@@ -310,4 +311,33 @@ public void testCreateRecordSent2(){
...
+        getWait().until(ExpectedConditions.elementToBeClickable(
+                getDriver().findElement(createButtonLocator))).click();
+
+        TestUtils.scrollClick(getDriver(), getDriver().
+                findElement(By.id("pa-entity-form-save-btn")));
```
Comment (AlexSharon) [inline src/test/java/EntityChevronTest.java:325]: and also here: SAVE_BUTTON = By.id("pa-entity-form-save-btn");

File: src/test/java/EntityChevronTest.java [hunk 3]
```diff
@@ -310,4 +311,33 @@ public void testCreateRecordSent2(){
         WebElement actualResult2 = getDriver().findElement(By.xpath("//tr/td[2]"));
         Assert.assertEquals(actualResult2.getText(),expectedResult);
     }
+
+    @Test
+    public void testChevronDeletePermanentlyRecord() {
```
Comment (AlexSharon) [inline src/test/java/EntityChevronTest.java:316]: I didn't find a corresponding Trello card in the description of PR. Please add.

**Review by AlexSharon — COMMENTED** [review]
Great job. Please, refer to the comments.
There are new tests in two classes in a single PR. I think we were suggested to have one PR per individual Trello card.

**Review by olhakovtun — APPROVED** [review]

**Pull request merged**